### ISSUE TITLE
Parse Error : syntax error, unexpected ';', expecting T_STRING on line 7   5 |   6 |>  7 | use ;     |    ^   8 | use Illuminate\Database\Eloquent\Model;   9 | use Illuminate\Database\Eloquent\Factories\HasFactory;  10 |

### ISSUE THREAD

**Issue by singhpk89** [VemtoOrg/vemto2-issues#104]
**Error Message:**
Parse Error : syntax error, unexpected ';', expecting T_STRING on line 7
   5 |
   6 |
>  7 | use ;
     |    ^
   8 | use Illuminate\Database\Eloquent\Model;
   9 | use Illuminate\Database\Eloquent\Factories\HasFactory;
  10 |

**Stack:**
```
SyntaxError: Parse Error : syntax error, unexpected ';', expecting T_STRING on line 7
   5 |
   6 |
>  7 | use ;
     |    ^
   8 | use Illuminate\Database\Eloquent\Model;
   9 | use Illuminate\Database\Eloquent\Factories\HasFactory;
  10 |
    at vt.raiseError (file:///C:/Program%20Files/Vemto/resources/app.asar/renderer/assets/index.3863d8c7.js:283:175)
    at vt.error (file:///C:/Program%20Files/Vemto/resources/app.asar/renderer/assets/index.3863d8c7.js:283:830)
    at vt.expect (file:///C:/Program%20Files/Vemto/resources/app.asar/renderer/assets/index.3863d8c7.js:284:414)
    at vt.read_namespace_name (file:///C:/Program%20Files/Vemto/resources/app.asar/renderer/assets/index.3863d8c7.js:272:37618)
    at vt.read_use_declaration (file:///C:/Program%20Files/Vemto/resources/app.asar/renderer/assets/index.3863d8c7.js:272:38451)
    at vt.read_use_statement (file:///C:/Program%20Files/Vemto/resources/app.asar/renderer/assets/index.3863d8c7.js:272:38029)
    at vt.read_top_statement (file:///C:/Program%20Files/Vemto/resources/app.asar/renderer/assets/index.3863d8c7.js:279:1527)
    at vt.read_top_statements (file:///C:/Program%20Files/Vemto/resources/app.asar/renderer/assets/index.3863d8c7.js:279:907)
    at vt.read_namespace (file:///C:/Program%20Files/Vemto/resources/app.asar/renderer/assets/index.3863d8c7.js:272:36852)
    at vt.read_start (file:///C:/Program%20Files/Vemto/resources/app.asar/renderer/assets/index.3863d8c7.js:272:36534)
```

**How to reproduce:**
1. 
2. 
3. 

**Expected behavior:**

**Actual behavior:**

**Additional context:**

---

Vemto 2.0.8
win32 10.0.22631
Locale: en-US
App version: 2.0.8

**Comment by TiagoSilvaPereira** [comment]
@singhpk89 thanks for reporting

**Comment by TiagoSilvaPereira** [comment]
@singhpk89 could you please check if the model namespace is empty?

**Comment by TiagoSilvaPereira** [comment]
Fixed on 2.0.11
